admin_manual/configuration_server/occ_command.rst

```diff
@@ -72,18 +94,220 @@ this example for the ``maintenance:mode`` command::
+ ca.js             fi.json     ml_IN.json  ta_LK.js
+ ca.json           fil.js      ml.js       ta_LK.json
+ ca@valencia.js    fil.json    ml.json     te.js
+ ca@valencia.json  fr_CA.js    mn.js       te.json
+ cs_CZ.js          fr_CA.json  mn.json     tg_TJ.js
+ cs_CZ.json        fr.js       mr.js       tg_TJ.json
+ cy_GB.js          fr.json     mr.json     th_TH.js
+ cy_GB.json        fy_NL.js    ms_MY.js    th_TH.json
+ da.js             fy_NL.json  ms_MY.json  tl_PH.js
+ da.json           gl.js       mt_MT.js    tl_PH.json
+ de_AT.js          gl.json     mt_MT.json  tr.js
+ de_AT.json        gu.js       my_MM.js    tr.json
+ de_CH.js          gu.json     my_MM.json  tzm.js
+ de_CH.json        he.js       nb_NO.js    tzm.json
+ de_DE.js          he.json     nb_NO.json  ug.js
+ de_DE.json        hi_IN.js    nds.js      ug.json
+ de.js             hi_IN.json  nds.json    uk.js
+ de.json           hi.js       ne.js       uk.json
+ el.js             hi.json     ne.json     ur.js
+ el.json           hr.js       nl.js       ur.json
+ en_GB.js          hr.json     nl.json     ur_PK.js
+ en_GB.json        hu_HU.js    nn_NO.js    ur_PK.json
+ en_NZ.js          hu_HU.json  nn_NO.json  uz.js
+ en_NZ.json        hy.js       nqo.js      uz.json
+ en@pirate.js      hy.json     nqo.json    vi.js
+ en@pirate.json    ia.js       oc.js       vi.json
+ eo.js             ia.json     oc.json     
+ eo.json           id.js       or_IN.js    yo.js
+ es_AR.js          id.json     or_IN.json  yo.json
+ es_AR.json        ignorelist  pa.js       zh_CN.js
+ es_BO.js          io.js       pa.json     zh_CN.json
+ es_BO.json        io.json     pl.js       zh_HK.js
+ es_CL.js          is.js       pl.json     zh_HK.json
+ es_CL.json        is.json     pt_BR.js    zh_TW.js
+ es_CO.js          it.js       pt_BR.json  zh_TW.json
+ es_CO.json        it.json     pt_PT.js
+ es_CR.js          ja.js       pt_PT.json
+ 
+LDAP Commands
+-------------
+
+You can run the following LDAP commands with ``occ``.
+
+Search for an LDAP user, using this syntax::
+
+ $ sudo -u www-data php occ ldap:search [--group] [--offset="..."] 
+ [--limit="..."] search
+
+This example searches for usernames that includes "rob"::
+
+ $ sudo -u www-data php occ ldap:search rob
+ 
+Check if an LDAP user exists. This works only if the ownCloud server is 
+connected to an LDAP server::
+
+ $ sudo -u www-data php occ ldap:check-user robert
+ 
+You can see your whole LDAP configuration, or the configuration for a single 
+configID::
+
+ $ sudo -u www-data php occ ldap:show-config
+ $ sudo -u www-data php occ ldap:show-config s01
+ 
+The ``ldap:set-config`` command is for manipulating configurations, like this 
+example that sets search attributes::
+ 
+ $ sudo -u www-data php occ ldap:set-config s01 ldapAttributesForUserSearch 
+ "cn;givenname;sn;displayname;mail"
+ 
+``ldap:test-config`` tests whether your configuration is correct and can bind to 
+the server::
+
+ $ sudo -u www-data php occ ldap:test-config ""
```

Comment:
Again, why does it do that? Just mentioning that it returns an empty string without saying why is also confusing.